### DISCUSSION TITLE
Carla not building from source on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by 5dollarperson** [carla-simulator/carla#8049] [General]
Hello, I have cloned this repository, and I am attempting to build this project from source. 
I am following directions from this link:
https://carla.readthedocs.io/en/latest/build_linux/

after cloning carla's branch of Unreal Engine, The step:
./Setup.sh && ./GenerateProjectFiles.sh && make

I receive this error:
Updating dependencies:   0% (0/63398)...
Failed to download 'http://cdn.unrealengine.com/dependencies/UnrealEngine-16118673/01acf3604f04c7b14a01645a77f5950f68863971': Error: TrustFailure (The authentication or decryption has failed.) (WebException)
ERROR building certificate chain: System.Threading.ThreadAbortException: Thread was being aborted.

---rest is omited---

Does anyone have any insight as to what I am doing wrong in this step of the build process

**Comment by fireofearth** [comment]
The solution that worked for me was to install `mono-runtime`. See:
https://github.com/carla-simulator/carla/issues/420#issuecomment-2275252760
